src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Layout/ManageNavMenu.razor

```diff
@@ -0,0 +1,37 @@
+﻿@using Microsoft.AspNetCore.Identity;
+@using BlazorWeb_CSharp.Data;
+
+@inject SignInManager<ApplicationUser> SignInManager;
+
+<ul class="nav nav-pills flex-column">
+    <li class="nav-item">
+        <NavLink id="profile" class="nav-link" href="/Account/Manage" Match="NavLinkMatch.All">Profile</NavLink>
```

Comment:
```suggestion
        <NavLink id="profile" class="nav-link" href="Account/Manage" Match="NavLinkMatch.All">Profile</NavLink>
```

No root-relative URLs :)

Reply:
Also, what are the `id`s for?

Reply:
These were copied from the Razor Pages implementation. We could probably remove them.